rpcs3/Emu/Cell/Modules/cellMic.cpp

```diff
@@ -34,161 +401,350 @@ s32 cellMicInit()
 {
 	cellMic.notice("cellMicInit()");
 
-	const auto micThread = fxm::make<mic_thread>("Mic Thread");
-	if (!micThread)
+	auto mic_t = g_idm->lock<mic_thread>(id_new);
```

Comment:
This abbreviation is used all over the new code and kinda bugs me since the _t suffix usually denotes a type. Why not just mic_thread or even mic_thr? Its a nitpick but its quite triggering to anyone who has worked with C for a long time.

Reply:
You're completely right, mic_thr it is!